src/utils/archive.ts

```diff
@@ -199,7 +200,59 @@ export async function createPublicDrive(
     txs: [driveTx, rootFolderTx]
   };
 }
+export async function sendArDriveFee(
```

Comment:
Add whitespace and docblocks

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)